jax/experimental/jax2tf/tests/correctness_stats.py

```diff
@@ -109,6 +109,14 @@ def _to_np_dtype(dtype) -> NpDType:
       # experimental_compile=True breaks for complex types.
       tf_unimpl(np_dtype)
 
+  if prim is lax_linalg.eig_p:
+    tf_unimpl(additional_msg=("this is a problem only in compiled mode "
```

Comment:
Does this mean that in compiled mode `eig` will not work? Does JAX have custom calls for CPU and GPU?

Reply:
Yes, the implementations are provided in `jaxlib.lapack.pyx` (`*geev` functions).

Reply:
I suggested adding a comment, or perhaps even text in the message saying that XLA does not support this case.

Reply:
I should have said implementation* (this is actually only on CPU). I will add it.